projboard/views.py

```diff
@@ -1,5 +1,19 @@
 from django.shortcuts import render
+from .models.article import Article
 
 
 def board(request):
     return render(request, 'projboard/board.html', {})
+
+
+def search(request):
+    title = ""
+    articles = []
+    if request.method == 'POST':
+        title = request.POST['title']
+        if title != '':
```

Comment:
What happens if title != '' ?

Reply:
i just wanted to prevent search articles , if the user hit search button by mistake without title input that gonna get him all articles in db .

Reply:
Is it the right way? 
Do you expect to have this query? maybe worth to notify the user? or prevent it? 
If not, don't you think you need to test it?

Reply:
it's not the right way , i should invilve a third party right there JS .

Reply:
or just use reqiure in html

Reply:
Can you return a message in the context once there is empty search? 
At least prepare and track it in the test?

You could also use the modal, see the latest talk for html